terraform/ec2/linux/install-efs-utils.sh

```diff
@@ -0,0 +1,32 @@
+# install amazon EFS client on linux, agnostic to yum, apt, or zypper + rpm
```

Comment:
**Nit pick**: Amazon License though

Reply:
Could you clarify what you mean?

Reply:
Adding Amazon License when creating new files though
```
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: MIT
```

Reply:
For every script based on my knowledge.